crates/red_knot_python_semantic/src/semantic_index/use_def.rs

```diff
@@ -326,6 +330,16 @@ impl<'db> UseDefMap<'db> {
         self.bindings_iterator(self.public_symbols[symbol].bindings())
     }
 
+    #[track_caller]
```

Comment:
This probably doesn't need a `[track_caller]` or is it possible that `bindings_iterator` panics? I otherwise don't see a method that might panic.

Reply:
Done